types/node-fetch/node-fetch-tests.ts

```diff
@@ -1,6 +1,7 @@
 import { Agent } from "http";
-import fetch, { AbortError, Blob, FetchError, Headers, Request, RequestInit, Response } from "node-fetch";
+import fetch, { AbortError, Blob, FetchError, Headers, Request, RequestInit, Response, HeaderInit, HeadersInit } from "node-fetch";
```

Comment:
It's not clear to me why this isn't failing CI; I would have thought that `export = fetch` would force `import fetch = require("node-fetch")` in the test. Maybe that message is being masked by the other errors.

Reply:
I'm also (and still) a little puzzled about what's going on with form-data; maybe I need to update its types, too (sort of the reverse of what you're confused about: would not expect form-data's behavior).

But AFAIK, `export =` is preferred/expected to fix default exports for node16/nodenext. See https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/FalseExportDefault.md

Reply:
`export =` means "the package wrote `module.exports =`". (You can get away with not doing `export =` when it's just an object of many named things.)

To be clear, I'm not looking at `form-data`'s import here, I'm looking at `node-fetch`. I'm meaning that I would have thought it would fail https://github.com/microsoft/DefinitelyTyped-tools/blob/main/packages/eslint-plugin/docs/rules/no-import-default-of-export-equals.md

Reply:
Sorry I think what I said was unclear — maybe just ignore my form-data comment in the last comment :).

I have a suspicion as to why it works, but I'd rather @andrewbranch (or someone else) weigh in before I say something that's close but not quite right.

Reply:
Updated the description to include a quick reference to include the relevant export sections of node-fetch's published source.

Reply:
I think it’s not failing the rule because the default import isn’t resolving to the `export =`. It’s resolving to the `export { _default as default }`.